crates/sui-framework/packages/bridge/sources/committee.move

```diff
@@ -160,17 +176,18 @@ module bridge::committee {
             let registration = CommitteeMemberRegistration {
                 sui_address: sender,
                 bridge_pubkey_bytes,
+                bridge_network_pubkey_bytes,
```

Comment:
I _believe_ that's ok, as I don't see any utility for an attacker to register with someone else's pubkey.

1. We check that there is no key duplication within the committee, so you won't be able to impersonate an active member.
2. If you add an arbitrary pubkey (one that doesn't exist), you just footgun yourself, since your requests will not have the elevated trust assumptions that this could provide. 

re: doc - I do not, but I can work on one. There is a long discussion thread externally, with lots of thoughts and opinions, and 3 proposed schemes. All of them would involve introducing a new key. I will try and organize them into a doc today.